Jenkins/update_ci_win_docker_image.ps1

```diff
@@ -0,0 +1,83 @@
+
+$ErrorActionPreference = "Stop"
+
+$globalVariables = (Resolve-Path "$PSScriptRoot\..\global-variables.ps1").Path
+$ciUtils = (Resolve-Path "$PSScriptRoot\..\Modules\CIUtils").Path
+
+Import-Module $ciUtils
+. $globalVariables
+
+$WORKSPACE="$env:WORKSPACE"
+$DOCKER_HUB_USER="$env:DOCKER_HUB_USER"
+$DOCKER_HUB_USER_PASSWORD="$env:DOCKER_HUB_USER_PASSWORD"
+$DOCKERFILE_PATH="$PSScriptRoot\ci-image-Dockerfile"
```

Comment:
Change this to `$DOCKERFILE_PATH = Join-Path $PSScriptRoot "ci-image-Dockerfile"`